homeassistant/components/evohome/__init__.py

```diff
@@ -4,14 +4,15 @@
 """
 from __future__ import annotations
 
-from datetime import datetime as dt, timedelta
+from collections.abc import Coroutine
+from datetime import datetime as dt, timedelta as td
```

Comment:
It is not common to alias these.

```suggestion
from datetime import datetime, timedelta
```